examples/purchase-tester/src/main/res/layout/fragment_login.xml

```diff
@@ -61,6 +61,13 @@
                 app:layout_constraintEnd_toEndOf="parent"
                 app:layout_constraintStart_toStartOf="parent" />
 
+        <com.google.android.material.button.MaterialButton
+                android:id="@+id/reset_sdk_button"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:text="Reset SDK"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toEndOf="parent" />
```

Comment:
I've only added it to the login fragment. Originally I thought about adding it to the overview fragment as well, but that screen was already pretty crowded and the login fragment is already accessible anyway by logging out.